pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/service/auth/AuthenticationService.java

```diff
@@ -69,6 +73,9 @@ public Response getToken(Map<String, String> authRequest){
             //Try to match
             user = matchingService.matchTokenToUser(userInfo);
             if (user == null){
+                if (JAXRSConfiguration.deniedEmailEnabled.startsWith("true")) {
+                    mailService.sendDeniedAccessEmail(userInfo);
+                }
                 throw new NotAuthorizedException("No user matching user_id " + userId + " present in database");
```

Comment:
seriously? if deniedEmailEnabled doesn't start with "true" we should throw a not authorized exception?